api/v6/configuration.thrift

```diff
@@ -0,0 +1,20 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+include "shared.thrift"
+
+namespace py Configuration_v6
+namespace js codeCheckerConfiguration_v6
+
+service codeCheckerConfiguration {
+
+    // Returns Global notification string.
+    string getNotification()
```

Comment:
Maybe a bit more specific naming would be better `getNotificationBannerText`. Notification is a a bit too wide, maybe we will introduce admin notification or user notification later. @csordasmarton what do you thin?